src/test/java/org/opensrp/web/rest/ProductCatalogueResourceTest.java

```diff
@@ -97,7 +97,7 @@ public void testGetAll() throws Exception {
 		productCatalogue.setUniqueId(1l);
 		List<ProductCatalogue> productCatalogues = new ArrayList<>();
 		productCatalogues.add(productCatalogue);
-		when(productCatalogueService.getProductCatalogues(any(ProductCatalogueSearchBean.class), anyString()))
+		when(productCatalogueService.getProductCatalogues(any(ProductCatalogueSearchBean.class), any(Integer.class), anyString()))
```

Comment:
We should add a new test case for the new method. The old method is Deprecated for removal but still in use. Once we remove it then we can delete the unit test